docs/new_alg.rst

```diff
@@ -0,0 +1,126 @@
+Create Customized Algorithms
+======================
+
+Goal of this tutorial:
+
+- Learn how to implement your own algorithms.
+
+
+Overview
+-----------
+
+To build a new algorithm, you need to inherit class ``Algorithm`` from ``parl.core.fluid.algorithm`` or ``parl``
+and implement three basic functions: ``sample``, ``predict`` and ``learn``.
+
+
+Methods
+-----------
+
+- ``__init__``
+
+  As algorithms update weights of the models, this method needs to define ``self.model``.
```

Comment:
this method needs to define some models inherited from``parl.Model``, like ``self.model`` in this example.